tests/MongoDB.Driver.Legacy.Tests/Operations/BulkWriteOperationTests.cs

```diff
@@ -1127,18 +1146,21 @@ public void TestUpsertWithOneMatchingDocument(bool ordered)
         [InlineData(true)]
         public void TestW0DoesNotReportErrors(bool ordered)
         {
+            var server = GetServerFromDisposableClient();
```

Comment:
Do this instead:

    var server = LegacyTestConfiguration.GetServer(retryWrites: false);

Reply:
Done